src/observability/summarizer.go

```diff
@@ -73,6 +73,27 @@ func extractNetworkInfoFromSystemLog(netLog pb.Log, pods []types.Pod, services [
 	return ip, destNs, destLabel, port, bindPort, bindAddress, protocol, nwrule, nil
 }
 
+func extractSyscallInfoFromSystemLog(sysCallLog pb.Log, pods []types.Pod, services []types.Service) (string, string, string, string, error) {
+	var ParentProcess, ChildProcess, Syscall, Parameters string = "", "", "", ""
+	err := errors.New("not a valid Syscall")
+
+	if sysCallLog.Operation == types.OperationTypeSyscall {
+		ParentProcess = sysCallLog.ParentProcessName
```

Comment:
Why are we using ParentProcessName and ProcessName instead of Source and Resource from logs.

Reply:
We only need ParentProcessName and ProcessName for syscall and resource is not necessary.
FYI: we have edited the "ParentProcessName -> Source" when we are working "summary data".